Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -88,6 +105,13 @@ private int Start(string file, bool debug)
             return VSConstants.S_OK;
         }
 
+        private static bool CheckUseNewChromeDebugProtocolOption()
+        {
+            var optionString = NodejsDialogPage.LoadString(name: "DebugProtocol", cat: "ChromeDebuggingVersion");
```

Comment:
Can you use `cat: "Debugging"` so all debugging options are under the same sub-key